### PR TITLE
wall time - replaces dorms and maint bars with walls where applicable

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
all the dorms and maint bars i could find are walls now
## Why It's Good For The Game
erp man bad
## Changelog
:cl:
del: dorms man bad
add: wall man good
/:cl:

### CODE REVIEW

**Comment by silicons** [comment]
BITCH

**Comment by Sishen1542** [comment]
you have my sword

**Comment by BlackMajor** [comment]
Kevinz why

**Comment by silicons** [comment]
okay that was funnier on paper than in practice

**Comment by Arturlang** [comment]
i ded PR

**Review by BurgerLUA — APPROVED** [review]

**Review by Arturlang — APPROVED** [review]

**Review by Sishen1542 — CHANGES_REQUESTED** [review]
can i get uhhhhhh

**Review by BlackMajor — APPROVED** [review]
This is an excellent PR.
You see with the nature of this being an ERP server the playerbase has always wished they could fuck wildly in the hall like animals. With the removal of private scene rooms, it becomes mandated that nowhere is safe to fuck. Lacking this safety, the people come out into the halls, spreading their corruption.
I dream of a server where I can start my ERP without having to move from my spawn tile.

File: _maps/map_files/OmegaStation/OmegaStation.dmm [hunk 1]
```diff
@@ -82322,14 +82201,14 @@ atV
...
+awQ
+awQ
 awQ
 aAJ
 aBT
 awQ
-aDI
-aER
+awQ
+awQ
```
Comment (Sishen1542) [inline _maps/map_files/OmegaStation/OmegaStation.dmm:82211]:
```suggestion
OwO
```

Comment (Hatterhat) [reply _maps/map_files/OmegaStation/OmegaStation.dmm:82211]: no

File: _maps/map_files/OmegaStation/OmegaStation.dmm [hunk 2]
```diff
@@ -22738,7 +22611,9 @@
 /turf/open/floor/engine,
 /area/engine/engineering)
 "aOa" = (
-/obj/machinery/power/supermatter_crystal/shard/engine,
+/obj/machinery/nuclearbomb/selfdestruct{
+	r_code = "11111"
```
Comment (Sishen1542) [inline _maps/map_files/OmegaStation/OmegaStation.dmm:22615]:
```suggestion
	r_code = "voreisbad"
```

Comment (Hatterhat) [reply _maps/map_files/OmegaStation/OmegaStation.dmm:22615]: bitch i CANNOT put numbers into the nuke code

Comment (Hatterhat) [reply _maps/map_files/OmegaStation/OmegaStation.dmm:22615]: i meant letters

**Comment by Mazianni** [comment]
excellent

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
You missed, bar back room, cap room, sec, perma, medical dorms, RD office, CMO office and cargo crate storage as well as miner dorms. Please wall theses zones off as well

**Review by Arturlang — APPROVED** [review]
Testmerge please

**Comment by Arturlang** [comment]
Hmm, what about grilles into other private areas we cant get rid of, not glass since that can be spraypainted easily.

File: _maps/map_files/OmegaStation/OmegaStation.dmm (hunk 2 above)
Comment (Arturlang) [reply _maps/map_files/OmegaStation/OmegaStation.dmm:22615]: Code it so you can, cant be that hard, right?

**Comment by ghost** [comment]
speedmerge

**Comment by Ludox235** [comment]
How is this not merged yet

**Comment by Fractious** [comment]
What the fuck.

**Comment by lolman360** [comment]
>chad rp explorer main is removing erp zones

godspeed

**Comment by Hatterhat** [comment]
F

**Comment by nik707** [comment]
reopen this pls orange man bad

**Pull request closed without merging**